### ISSUE TITLE
Scope out new event management Admin API

### ISSUE THREAD

**Issue by mkurapov** [interledger/rafiki#1521]
Instead of using `eventId` in order to add/withdraw/deposit liquidity, we should consider using `outgoingPaymentId` and `incomingPaymentId` to move around liquidity.

**Comment by mkurapov** [comment]
CC @sabineschaller

**Comment by sabineschaller** [comment]
Yes, I agree. Event based always seemed weird to me. Do we then still need the view of all events?

**Comment by mkurapov** [comment]
@sabineschaller reposting my response in #1503:

> We still would like the option to be able to view all of the different events that have been sent. The issue https://github.com/interledger/rafiki/issues/1521 above is more concerned with how and by which parameter we want to track liquidity movement.
